crates/utility-server/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "utility-server"
```

Comment:
I don't agree with this being called `utility-server`. Should be called `http-server` (if we're going to use some generic name) or `health-check-server` if we are being specific. `utility` does not say anything about what this does.

Reply:
We discussed that exact same thing with @eyusufatik :D The name for him `http-server` was too generic but I also believe it is a good name since it is a different protocol and the `rpc-server` name is based on another protocol `jsonrpc` which makes sense.

What are your final thoughts on naming @eyusufatik ?

I would also like your @rakanalh @rakanalh @kpp  opininon on health-checking logic for prover nodes, current implementation checks if there is a new soft confirmation after 3 seconds, but since that logic won't work with prover we need either new method for prover or a new health-check logic entirely.